game/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2018"
 [features]
 default = ["built", "ezgui/glium-backend", "reqwest", "webbrowser"]
 wasm = ["ezgui/wasm-backend"]
+glow = ["built", "ezgui/glow-backend", "reqwest", "webbrowser"]
```

Comment:
Is this useful? Otherwise I can remove.

Note, I was able to get the Glow build to compile, but the screen was blank. This was the same behavior as master. If there's some other way to properly build glow, let me know and I can take a deeper look.

Reply:
Yeah, let's keep this, so we can pass `--features glow` to cargo. Previously, I was just commenting out stuff temporarily to test glow.